pkg/proxy/winkernel/proxier.go

```diff
@@ -1138,16 +1157,26 @@ func (proxier *Proxier) syncProxyRules() {
 		containsPublicIP := false
 		containsNodeIP := false
 
+		allEndpointsTerminating := proxier.isAllEndpointsTerminating(svcName, svcInfo.localTrafficDSR)
```

Comment:
We need to scope this to only services of Type LoadBalancer. Active connections to ClusterIP services are not impacted by terminating endpoints today in Windows.

Reply:
Addressed your comments.